lib/location/suite_location.js

```diff
@@ -0,0 +1,11 @@
+export class SuiteLocation {
+  #locationPath;
```

Comment:
let's rename it to "path" to avoid redundant naming with the object (already has location in its name)